src/views/NotFound.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-container class="fill-height">
+    <v-responsive class="d-flex align-center text-center fill-height ">
+      <v-sheet
+          border
+          rounded
+          :height="300"
+          :width="800"
+          class="frosted px-8 py-6 accent-border accent-text mx-auto "
+      >
```

Comment:
@cesarvh - it seems that your "continuation indentation" rule is four spaces. I think it should be two spaces. You can enforce such rule via `.eslintrc.js`